docs/testing.rst

```diff
@@ -32,3 +38,22 @@ Once the Makefile has been run, you can run ``make test`` to run the test suite.
      mv autopush/tests/test_logging.py{.hold,}
 
 This script will cause the integration and logging tests to not run.
+
+.. _test-with-firefox:
+
+Firefox Testing
+===============
+
+To test a locally running Autopush with Firefox, you will need to edit
+several config variables in Firefox.
+
+1. Open a New Tab.
+2. Go to ``about:config`` in the Location bar and hit Enter, accept the disclaimer
+   if it's shown.
+3. Search for ``dom.push.serverURL``, make a note of the existing value (usually
+   ``wss://push.services.mozilla.com/``.
+4. Double click the entry and change it to ``ws://localhost:8080/``.
+5. Right click in the page and choose ``New -> Boolean``, name it
+   ``dom.push.testing.allowInsecureServerURL`` and set it to ``true``.
```

Comment:
A couple of additions, maybe for a follow-up:

* On Android, setting `dom.push.debug` to `true` enables debug logging via `adb logcat`. (There's no harm in setting it on Desktop; it just won't do anything).
* It might also be a good idea to set `dom.push.loglevel` to `"debug"`; that'll dump everything sent over the wire to the Browser Console (Tools > Web Developer > Browser Console).